src/System.Net.Primitives/tests/PalTests/SocketAddressPalTest.cs

```diff
@@ -188,5 +188,15 @@ public void IPv6AddressAndPort_Get_Set(byte[] address, uint scope, ushort port)
             Assert.Equal(scope, actualScope);
             Assert.Equal(port, SocketAddressPal.GetPort(buffer));
         }
+
+        // This is platform specific test. So far Windows, Linux and OSX use same layout.
```

Comment:
nit: This comment is confusing.  This test is not "platform-specific".  It is defined as a "Fact" and will run for any platform.  I would delete this comment.